bundles/org.openhab.binding.teleinfo/README.md

```diff
@@ -146,40 +146,40 @@ The following channels are available on all Linky telemeters in standard TIC mod
-| commonLSMGroup#dpm*X*Date       | `DateTime`                | Date of DPM*X*, *X* in {1,2,3}                                              |
-| commonLSMGroup#fpm*X*Date       | `DateTime`                | Date of FPM*X*, *X* in {1,2,3}                                              |
-| commonLSMGroup#relais*X*        | `Switch`                  | state of relais *X*, *X* in {1,...,8}                                       |
+| commonLSMGroup#dpm_X_Date       | `DateTime`                | Date of DPM_X_, _X_ in {1,2,3}                                              |
+| commonLSMGroup#fpm_X_Date       | `DateTime`                | Date of FPM_X_, _X_ in {1,2,3}                                              |
+| commonLSMGroup#relais_X_        | `Switch`                  | state of relais _X, X_ in {1,...,8}                                       |
 
 #### Three phase only channels
 
 These channels are available on the following telemeters:
 
-* lstm_electricitymeter
-* lsmt_prod_electricitymeter
+- lstm_electricitymeter
+- lsmt_prod_electricitymeter
 
 | Channel                                 | Type                      | Description                                                                       |
 |-----------------------------------------|---------------------------|-----------------------------------------------------------------------------------|
-| threePhasedLSMGroup#irms*X*             | `Number:ElectricCurrent`  | RMS Current on phase *X*, *X* in {2,3}                                            |
-| threePhasedLSMGroup#urms*X*             | `Number:Potential`        | RMS Voltage on phase *X*, *X* in {2,3}                                            |
-| threePhasedLSMGroup#umoy*X*             | `Number:Potential`        | Mean Voltage on phase *X*, *X* in {2,3}                                           |
-| threePhasedLSMGroup#sinsts*X*           | `Number:Power`            | Instantaneous withdrawn apparent power on phase *X*, *X* in {1,2,3}               |
-| threePhasedLSMGroup#smaxsn*X*           | `Number:Power`            | Maximum withdrawn apparent power of the day on phase *X*, *X* in {1,2,3}          |
-| commonLSMGroup#umoy*X*Date              | `DateTime`                | Timestamp of UMOY*X* value, *X* in {2,3}                                          |
-| threePhasedLSMGroup#smaxsn*X*Minus1     | `Number:Power`            | Maximum withdrawn apparent power on the previous day on phase *X*, *X* in {1,2,3} |
-| threePhasedLSMGroup#smaxs*X*nDate       | `DateTime`                | Timestamp of SMAXSN*X* value, *X* in {1,2,3}                                      |
-| threePhasedLSMGroup#smaxsn*X*Minus1Date | `DateTime`                | Timestamp of SMAXSN*X*-1 value, *X* in {1,2,3}                                    |
+| threePhasedLSMGroup#irms_X_             | `Number:ElectricCurrent`  | RMS Current on phase _X, X_ in {2,3}                                            |
```

Comment:
Here and in the next rows: I think intention here was to underline each X individually without the comma?

Reply:
This is on purpose as is now.
I have tried several options to make this pass markldown linting and also keep the existing intention from the old markup.

Reply:
I would like to resolve this, if you don't see any bigger issue with this.

From the rich diff here in the PR it looks quite good.
We can check it anyway after merging and see how vuepress will render this in the `next` homepage.

Reply:
One thing though, can we have the alignment of this table as well as the previous/next tables restored?